Ecommerce/EcommerceWeb/wwwroot/js/user.js

```
@@ -0,0 +1,68 @@
+    dataTable = $('#tblData').DataTable({
+        "ajax": { url: '/admin/user/getall' },
+        "columns": [
+            { "data": "name", "width": "15%" },
+            { "data": "email", "width": "15%" },
+            { "data": "phoneNumber", "width": "15%" },
+            { "data": "company.name", "width": "15%" },
+            { "data": "role", "width": "15%" },
+            {
+                data: { id: "id", lockoutEnd: "lockoutEnd" },
+                "render": function (data) {
+                    let today = new Date().getTime();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'today' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)